SeriesCruncher/app/src/main/java/com/balazs_csernai/seriescruncher/utils/dialog/NetworkErrorDialogFragment.java

```diff
@@ -74,4 +74,77 @@ public void onClick(DialogInterface dialog, int which) {
     public String getRandomText(String[] texts) {
```

Comment:
Idea: wrap the string array initialization and the random getter

private String getRandomText(@ArrayRes int arrayResId)